unyt/_unit_lookup_table.py

```diff
@@ -195,7 +200,11 @@
             ),
         ),
         ("cal", (4.184, dimensions.energy, 0.0, r"\rm{cal}", True)),
-        ("BTU", (1055.0559, dimensions.energy, 0.0, r"\rm{BTU}", False)),
+        ("BTU", (J_per_BTU, dimensions.energy, 0.0, r"\rm{BTU}", False)),
+        ("MMBTU", (btu_per_mmbtu, dimensions.energy, 0.0, r"\rm{MMBTU}", False)),
+        ("therm", (btu_per_therm, dimensions.energy, 0.0, r"\rm{therm}", False)),
+        ("quad", (btu_per_quad, dimensions.energy, 0.0, r"\rm{quad}", False)),
```

Comment:
If anyone else is unfamiliar with these units, from wikipedia:

>Units kBtu are used in building energy use tracking and heating system sizing. Energy Use Index (EUI) represents kBtu per square foot of conditioned floor area. "k" stands for 1,000.

>The unit Mbtu is used in natural gas and other industries to indicate 1,000 BTUs.[[11]](https://en.wikipedia.org/wiki/British_thermal_unit#cite_note-EIA-13)[[12]](https://en.wikipedia.org/wiki/British_thermal_unit#cite_note-Price-14) However, there is an ambiguity in that the metric system (SI) uses the prefix "M" to indicate '[Mega-](https://en.wikipedia.org/wiki/Mega-)', one million (1,000,000). Even so, "MMbtu" is often used to indicate one million BTUs particularly in the oil and gas industry.[[13]](https://en.wikipedia.org/wiki/British_thermal_unit#cite_note-APS-15)

>Energy analysts accustomed to the metric "k" ('[kilo-](https://en.wikipedia.org/wiki/Kilo-)') for 1,000 are more likely to use MBtu to represent one million, especially in documents where M represents one million in other energy or cost units, such as MW, MWh and $.

>The unit '[therm](https://en.wikipedia.org/wiki/Therm)' is used to represent 100,000 BTUs.[[11]](https://en.wikipedia.org/wiki/British_thermal_unit#cite_note-EIA-13) A decatherm is 10 therms or one MMBtu (million Btu). The unit [quad](https://en.wikipedia.org/wiki/Quad_(unit)) is commonly used to represent one quadrillion (1015) BTUs.[[13]](https://en.wikipedia.org/wiki/British_thermal_unit#cite_note-APS-15)

Definitely awesome that there's some ambiguity about whether SI prefixes make sense or not. I agree with your choice to just not allow it.